internal/fsutil/create_test.go

```diff
@@ -15,13 +16,15 @@ import (
 )
 
 type createTest struct {
+	summary string
 	options fsutil.CreateOptions
-	hackopt func(c *C, dir string, options *fsutil.CreateOptions)
+	hackopt func(c *C, targetDir string, options *fsutil.CreateOptions)
 	result  map[string]string
 	error   string
 }
 
 var createTests = []createTest{{
+	summary: "Create a file under a new directory",
```

Comment:
```suggestion
	summary: "Create a file and its parent directory",
```
Let's be consistent, if you mention the parent directory here, let's also mention it in the other tests that use it.

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9).